dataactvalidator/config/sqlrules/d37_detached_award_financial_assistance_1.sql

```diff
@@ -0,0 +1,25 @@
+-- For new assistance awards (ActionType = A), the CFDA_Number must be active as of the ActionDate.
+-- This does not apply to correction records (those with CorrectionLateDeleteIndicator = C).
+-- If Archived Date <= Action Date <= Published Date, it passes validation (active).
+SELECT
+    row_number,
+    cfda_number,
+    action_type,
+    correction_late_delete_ind,
+    action_date
+FROM detached_award_financial_assistance AS dafa
+WHERE submission_id = {0}
+    AND dafa.action_type = 'A'
+    AND ((dafa.correction_late_delete_ind != 'C')
+        or (dafa.correction_late_delete_ind is null)
+    )
+    AND dafa.row_number IN (
+        SELECT DISTINCT dafa.row_number
+        FROM detached_award_financial_assistance AS dafa
+            JOIN cfda_program AS cfda
```

Comment:
The cfda_program table is not big, it has 2908 lines (compare to program_activity which has 19931 lines). I did add  'SELECT DISTINCT dafa.row_number' as @alburde1 recommended for all `JOIN` to speed up the validation.